picard/script.py

```diff
@@ -283,6 +284,13 @@ def eval(self, script, context=None, file=None):
             ScriptParser._cache[key] = self.parse(script, True)
         return ScriptParser._cache[key].eval(self)
 
+def enabled_tagger_scripts_texts():
+    """Returns an iterator over the enabled tagger scripts.
+    For each script, you'll get a tuple consisting of the script name and text"""
+    if not config.setting["enable_tagger_scripts"]:
+        return []
+    return [ (s_name, s_text) for _, s_name, s_enabled, s_text in config.setting["list_of_scripts"] if s_enabled and s_text]
```

Comment:
We generally don't use `_` as a discard-able variable in Picard since picard changes `_` to gettext as a builtin.

Reply:
Yes, please change it, otherwise the PR looks good to me.